packages/core/discovery.config.default.js

```diff
@@ -75,11 +75,11 @@ module.exports = {
   },
 
   // Production URLs
-  storeUrl: 'https://vtexfaststore.com',
-  secureSubdomain: 'https://secure.vtexfaststore.com',
-  checkoutUrl: 'https://secure.vtexfaststore.com/checkout',
-  loginUrl: 'https://secure.vtexfaststore.com/api/io/login',
-  accountUrl: 'https://secure.vtexfaststore.com/api/io/account',
+  storeUrl: 'https://homebrewqa.fast.store',
+  secureSubdomain: 'https://homebrewqa.fast.store',
```

Comment:
ps: Do you think we need to add a note here? Is the same as storeUrl because we are using single domain approach for this acc?

Reply:
done [here](https://github.com/vtex/faststore/pull/2802/commits/b824261866a9c1c89d5155ca251a39ced22d2d2e).